packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx

```diff
@@ -61,6 +62,24 @@ export const CreateTopicSection = () => {
     }
   };
 
+  const getCharacterCount = (delta) => {
+    if (!delta || !delta.ops) {
+      return 0;
+    }
+    return delta.ops.reduce((count, op) => {
+      if (typeof op.insert === 'string') {
+        const cleanedText = op.insert.replace(/\n$/, '');
+        return count + cleanedText.length;
+      }
+      return count;
+    }, 0);
+  };
+
+  useEffect(() => {
+    const count = getCharacterCount(descriptionDelta);
+    setCharacterCount(count);
+  }, [descriptionDelta]);
+
```

Comment:
it looks over complicated, especially that it worked some time ago. Could you try to revert this change and check if it works? 

https://github.com/hicommonwealth/commonwealth/commit/e6265d0a01495a4f70060c977d172ac245d41438#diff-0c09f2a69a0da3f96a9b614ad286aa003240b477153e110e3b34321d5d85fc5a

```
            <CWText type="caption">
              {descriptionDelta?.ops?.[0].insert?.length} / 250
            </CWText>
```

Reply:
@masvelio I did try that. The problem with this is insert adds a "/n" character, and so the length count is always +1 of what the actual count was. And if you deleted all of the characters there would still be a count of 1.

Reply:
thanks for checking this